src/main/java/org/apache/datasketches/kll/KllDoublesQuantileCalculator.java

```diff
@@ -167,5 +132,39 @@ private static void tandemMerge(
     }
   }
 
-}
+  double getQuantile(final double phi) { //phi is normalized rank [0,1].
```

Comment:
I would suggest naming the parameter "rank" instead of "phi", so that we wouldn't have to clarify in a comment

Reply:
Done.